### ISSUE TITLE
Encrypt passwords

### ISSUE THREAD

**Issue by schvabodka-man** [mopidy/mopidy-scrobbler#25]
Add possibility to encrypt your stuff. Using as is is very insecure

**Comment by jjok** [comment]
Which stuff do you mean?

**Comment by schvabodka-man** [comment]
Stuff in your mopidy config for scrobbler. You know, username and password for last.fm

**Comment by dz0ny** [comment]
But those are not publicly accessible, they are secured under system security model. Only users with read access should be able to read them, that's why we recommend running under a different user. 

https://docs.mopidy.com/en/latest/service/#running-as-a-service
https://en.wikipedia.org/wiki/File_system_permissions

**Comment by schvabodka-man** [comment]
Yes, you can do chmod 600, but the're still in plain text format

**Comment by marsjaninzmarsa** [comment]
Duplicate of #1.

**Comment by evamvid** [comment]
[I've submitted a pull request that would implement this by requiring the user to put the md5 of their password instead of the plaintext.](https://github.com/mopidy/mopidy-scrobbler/pull/27)

**Comment by evamvid** [comment]
#27

**Comment by jodal** [comment]
It's 2026, full disk encryption is everywhere, and everyone should use password managers and unique passwords for every service. In other words, your Last.fm or Libre.fm password is equivalent to any API key. We're not going to add support for encrypting passwords on disk.
